lib/src/facts/linux/dmi_resolver.cc

```diff
@@ -56,4 +71,124 @@ namespace facter { namespace facts { namespace linux {
         return value;
     }
 
+    void dmi_resolver::parse_dmidecode_output(data& result, string& line, int& dmi_type)
```

Comment:
I think it would make sense to test this function; that helps document our expectation of the output.